### ISSUE TITLE
[Feature Request] Full Screen compose, inbox, and reader

### ISSUE THREAD

**Issue by Pome-ro** [leggett/simplify#713]
Hey there, 

I deal in screenshots regularly, and I often include the "Original Size" because the text needs to be readable (I do tech support/systems support). These screenshots can sometimes bleed off the edge of the compose screen, making coposing an email difficult. It made me realize how much horizontal space is unutilized on the flanks of the inbox, full screen compose, and reader views within Gmail. I think at a minimum it would be great to have a "full screen" button for composing an email, that stretches the compose window horizontally and fills the screen. 

This may have been suggested before but I did some searching through the issue tracker and didn't come up with much.

**Comment by erikgeurts** [comment]
I agree, optionally being able to compose an email (or read one) in an area that uses the entire available width and height of the browser viewport would mean a lot to me.

**Comment by leggett** [comment]
Putting a screenshot inline in an email at original size is usually going to be a bad experience for the recipient as it will scroll off the screen for them too. Simplify fixes this in Gmail when viewing a message (but not in compose b/c I don't want to mask what you are sending) and keeps images from being wider than the message.

That said, there are several ways to achieve what you desire today:

**Two "just for this email" solutions:**

- Pop-out the email into a new window where I let the email (and any inline replies) be as large as the window
- Temporarily disable Simplify with [⌥S] on Mac or [Alt+S] on Win/Unix or the Simplify menu as Gmail does not restrict the width of the email (or the inline reply)

**An "for every email" solution:**

- Adjust the width of conversation view in Simplify settings. It can go up to full-screen. This width is also applied to the fullscreen composer (whether you use it in Gmail, in a new window with [⇧C], or in a new Tab with [D]

The only thing I can think to change or add with Simplify is let you maximize the width of a fullscreen composer for just one message (instead of having to change the max-width settings for all conversations). This might be best solved with my planned ⌘K menu that will give you quick access to Gmail actions and Simplify settings from anywhere in Gmail.

**Comment by leggett** [comment]
I meant to add: when I send high resolution screenshots, I attach them because of the above issue. I use filenames that align with my instructions (step_1_do_this_thing.png). 

Not only does this make it easier to read the instructions and if they happen to reply, it doesn't include the attached images in the quoted text (it will include any inline images in every reply which can also prevent an image from being attached to a reply as it is already near the 25MB limit). 

I wrote more about why I put a max-width on the conversation and how large inline images make things worse in this old blog post: https://on.simpl.fyi/p/simplify-v2-conversation-view
